lib/bootstrap_form/inputs/check_box.rb

```diff
@@ -65,16 +68,22 @@ def check_box_label_class(options)
       def check_box_wrapper_class(options)
         classes = []
         if options[:custom]
-          classes << "custom-control"
-          classes << (options[:custom] == :switch ? "custom-switch" : "custom-checkbox")
-          classes << "custom-control-inline" if layout_inline?(options[:inline])
+          classes << custom_check_box_wrapper_class(options)
         else
           classes << "form-check"
```

Comment:
For radio buttons you defined a `standard_radio_button_wrapper_class`. Did you want to define a `standard_check_box_wrapper_class`? I don't care which way you do it, but I thought I'd mention it in case you meant to and forgot.

Reply:
I was refactoring specifically to meet rubocop requirements, rather than necessarily to make them behave in the same way, hence I did it like this.

Reply:
OK. Thanks.